$NetBSD$

Prevents build failure due to -Ddouble=__grub_poision.  A build log excerpt is:

In file included from /usr/include/i386/ansi.h:37:0,
                 from /usr/include/machine/ansi.h:66,
                 from /usr/include/sys/ansi.h:35,
                 from /usr/include/stdarg.h:37,
                 from ../include/grub/misc.h:23,
                 [...]
/usr/include/sys/cdefs.h:558:41: error: expected ')' before 'x'

--- conf/Makefile.common.orig	2013-08-13 22:37:38.000000000 +0000
+++ conf/Makefile.common
@@ -46,8 +46,12 @@ endif
 
 if ! COND_emu
 CFLAGS_PLATFORM += -Dfloat=__grub_poision
+# NetBSD 6.99 declares a function with type double in /usr/include/sys/cdefs.h
+# and the latter gets eventually pulled in by grub/misc.h
+if ! COND_HOST_NETBSD
 CFLAGS_PLATFORM += -Ddouble=__grub_poision
 endif
+endif
 
 CPPFLAGS_DEFAULT = -DGRUB_FILE=\"$(subst $(srcdir)/,,$<)\"
 CPPFLAGS_DEFAULT += -I$(builddir)
